### PR TITLE
Adding support for rgw for stackanetes

### PR DESCRIPTION
**Pull request by PiotrProkop**

### CODE REVIEW

**Comment by PiotrProkop** [comment]
@DTadrzak please review.

File: config/rgw/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+ceph: dupa
```
Comment (DTadrzak) [inline config/rgw/defaults/main.yml]: seriously? without new line at the end of this file?

File: stackanetes/config/ceph.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from oslo_config import cfg
+
+
+CONF = cfg.CONF
+# TODO(DTadrzak): add help information
```
Comment (DTadrzak) [inline stackanetes/config/ceph.py]: That is not my comment.

File: stackanetes/k8s_instance.py [hunk 3]
```diff
@@ -91,6 +108,4 @@ def _manage_instance(self, cmd_type):
 
         cmd.extend([cmd_type, "-f", self.file_path])
         LOG.debug('{} instance {}'.format(cmd_type, self.service_name))
-        subprocess.call(cmd)
-
-
+        subprocess.call(cmd)
```
Comment (DTadrzak) [inline stackanetes/k8s_instance.py]: add new line at the end of this file

File: stackanetes/opts.py [hunk 4]
```diff
@@ -12,10 +12,12 @@
 
 from stackanetes.config import logging
 from stackanetes.config import stackanetes
+from stackanetes.config import ceph
```
Comment (DTadrzak) [inline stackanetes/opts.py]: alphabetical order

**Comment by DTadrzak** [comment]
Please adjust stackanetes_deployer to those changes.

**Comment by DTadrzak** [comment]
LGTM

**Pull request merged**